### PR TITLE
Need --exclude-libs,ALL for cling

### PR DESCRIPTION
**Pull request by isuruf**

### CODE REVIEW

**Comment by certik** [comment]
Will this PR allow us to test the `notebooks/symengine.ipynb` on Travis? I would like to have it tested on the CI.

**Comment by isuruf** [comment]
Added testing of cling notebooks. Ready for review

File: .travis.yml [hunk 1]
```diff
@@ -177,7 +182,6 @@ before_install:
 install:
   - source bin/install_travis.sh
 script:
-  - bin/travis_clang_format.sh
```
Comment (certik) [inline .travis.yml]: Why was this removed? Such a change should be documented in the git commit message.

Comment (isuruf) [reply .travis.yml]: This is a redundant line. It's already in line 82.

**Comment by isuruf** [comment]
Thanks for the review
